bedrock/settings/base.py

```diff
@@ -569,15 +564,6 @@ def lazy_email_backend():
                          'tl_PH', 'tr_TR', 'uk_UA', 'vi_VN', 'zh_CN',
                          'zh_HK', 'zh_TW']
 
-
-# FACEBOOK_TAB_URL is lazily evaluated because it depends on the namespace
-# and app ID settings in local settings.
```

Comment:
I think this is sufficient. I'm sure there are `FACEBOOK_*` settings in the SCL3 environments, but I don't see anything in the AWS ones, so this is probably only working some of the time anyway :( I'll go in and delete these settings from the SCL3 envs after this is deployed.